spec/integrations/sidekiq_spec.rb

```diff
@@ -29,6 +30,8 @@ def perform(value)
         expect(event["metaData"]["sidekiq"]["msg"]["args"]).to eq([-0])
         expect(event["metaData"]["sidekiq"]["msg"]["queue"]).to eq("default")
         expect(event["severity"]).to eq("error")
+        expect(event["app"]["type"]).to eq("sidekiq")
+        expect(event["device"]["runtimeVersions"]["sidekiq"]).to eq(::Sidekiq::VERSION)
```

Comment:
This should also have some version string sanity check as Sidekiq isn't faked/mocked